Makefile

```diff
@@ -12,5 +35,19 @@ updaterdk:
 	go get go.viam.com/rdk@latest
 	go mod tidy
 
+FFmpeg:
+	git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
```

Comment:
I like this but could we make ffmpeg a git submodule? That way we can keep track of which specific commit of FFmpeg we are building against throughout the commit history of this module.

Here is an example of how we did that in viam-cartographer:
https://github.com/viamrobotics/viam-cartographer/tree/main/viam-cartographer

https://github.com/viamrobotics/viam-cartographer/blob/main/Makefile#L51

LMK if you would like to pair on this